src/app/domain/accounts/models.py

```diff
@@ -26,7 +26,8 @@ class User(orm.DatabaseModel, orm.AuditColumns):
     is_superuser: Mapped[bool] = mapped_column(default=False)
     is_verified: Mapped[bool] = mapped_column(default=False)
     verified_at: Mapped[datetime | None]
-    joined_at: Mapped[datetime] = mapped_column(server_default=func.now())
+    created: Mapped[datetime] = mapped_column(server_default=func.now())
+    updated: Mapped[datetime] = mapped_column(server_default=func.now(), server_onupdate=func.now())
```

Comment:
It looks like they are actually already there.  I think we only need the additional columns on the schema file.  These should be removed.
```suggestion
```